_posts/patterns/2015-12-21-lists.md

```diff
@@ -0,0 +1,308 @@
+        <p class="item">Raised inside a geyser field, the port of Darkwell is home to vikings lead by Mrs. Chilson. This port wasn't built by a geyser field by accident, as it has rare resources, which is of great importance to the people of Darkwell and its success.</p>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Link-based Lists</h3>
+<div class="example-set">
+    <nav class="list-links">
+        <span class="item is-current"><a href="http://ux.edx.org">Home</a></span>
+        <span class="item"><a href="http://ux.edx.org">About</a></span>
+        <span class="item"><a href="http://ux.edx.org">Jobs</a></span>
+        <span class="item"><a href="http://ux.edx.org">Team</a></span>
+    </nav>
+
+    <nav class="list-links list-inline">
+        <span class="item is-current"><a href="http://ux.edx.org">Home</a></span>
+        <span class="item"><a href="http://ux.edx.org">About</a></span>
+        <span class="item"><a href="http://ux.edx.org">Jobs</a></span>
+        <span class="item"><a href="http://ux.edx.org">Team</a></span>
+    </nav>
+
+    <ul class="list list-links">
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Apples (Block-level links)</a>
+        </li>
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Apricots (Block-level links)</a>
+        </li>
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Bananas (Block-level links)</a>
+        </li>
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">List with Dividers</h3>
+<div class="example-set">
+    <ul class="list list-divided">
+        <li class="item">Apples</li>
+        <li class="item">Apricots</li>
+        <li class="item">
+            <a href="https://solidthreads.com/media/catalog/product/i/m/image_78.jpg">Bananas</a>
+        </li>
+        <li class="item">Blackberries</li>
+    </ul>
+
+    <ul class="list">
+        <li class="item">Apples</li>
+        <li class="item has-divider-bottom">Apricots (Manual Bottom Divider)</li>
+        <li class="item">
+            <a href="http://ux.edx.org/">Bananas</a>
+        </li>
+        <li class="item has-divider-top">Blackberries (Manual Top Divider)</li>
+    </ul>
+
+    <ul class="list list-divided">
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Apples (Block-level links)</a>
+        </li>
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Apricots (Block-level links)</a>
+        </li>
+        <li class="item has-block-link">
+            <a href="http://ux.edx.org/">Bananas (Block-level links)</a>
+        </li>
+    </ul>
+</div>
+
+<h3 class="hd-6 example-set-hd">Lists with Complex Content</h3>
+<div class="example-set">
+    <div class="list">
+        <div class="is-current item item-complex">
```

Comment:
Since you set an "is-current" class on this, I would have a little affordance in the visual display to reflect that state.